### PR TITLE
feat(layout-grid): parameterize layout grid

### PR DESCRIPTION
**Pull request by yeelan0319**
BREAKING CHANGE in css custom properties and sass variables.

### CODE REVIEW

**Comment by traviskaufman** [comment]
@yeelan0319 FYI that commit message needs to be formatted as `BREAKING CHANGE:` (note the `:`) and needs to specifically detail what each breaking change is and how current code can be updated to match the new code.

**Comment by yeelan0319** [comment]
I will look for some examples, I know it must be documented somewhere.

**Comment by yeelan0319** [comment]
I changed the commit message. PTAL

File: packages/mdc-layout-grid/README.md [hunk 1]
```diff
@@ -70,24 +68,35 @@ Behavior for grids containing direct children without the `mdc-layout-grid__cell
...
+#### CSS custom properties
+
+You can change the margins and gutters for a grid using the `--mdc-layout-grid-margin-#{$device}` and `--mdc-layout-grid-gutter-#{$device}` custom properties, respectively. This requires support for CSS custom properties on the end-user's browser.
 
 ```css
 .my-grid {
-  --mdc-layout-grid-margin: 40px;
-  --mdc-layout-grid-gutter: 16px;
+  --mdc-layout-grid-margin-desktop: 40px;
+  --mdc-layout-grid-gutter-tablet: 16px;
```
Comment (traviskaufman) [inline packages/mdc-layout-grid/README.md]: nit: perhaps we should also show `--mdc-layout-grid-gutter-phone` being used here?

Comment (yeelan0319) [inline packages/mdc-layout-grid/README.md]: Sure thing!

**Pull request merged**